app/javascript/angular/code/calculateBounds.js

```diff
@@ -1,28 +1,16 @@
-export const calculateBounds = (sensors, sessions) => {
-  const maxLat = [];
-  const minLat = [];
-  const maxLong = [];
-  const minLong = [];
+export const calculateBounds = (sensors, selectedSession) => {
   const sensor = sensors.anySelected();
-
   if (!sensor) return;
 
-  sessions.forEach(function(session) {
-    const stream = session.streams[sensor.sensor_name];
-    if (!stream) return;
-
-    maxLat.push(stream.max_latitude);
-    minLat.push(stream.min_latitude);
-    maxLong.push(stream.max_longitude);
-    minLong.push(stream.min_longitude);
-  });
-
-  var north = Math.max.apply(null, maxLat);
-  var south = Math.min.apply(null, minLat);
-  var west = Math.min.apply(null, minLong);
-  var east = Math.max.apply(null, maxLong);
-
-  if (!north) return;
+  const stream = selectedSession.streams[sensor.sensor_name];
+  if (stream) {
+    return {
+      north: stream.max_latitude,
+      east: stream.max_longitude,
+      south: stream.min_latitude,
+      west: stream.min_longitude
+    };
+  }
 
-  return { north, east, south, west };
+  return { north: -Infinity, east: -Infinity, south: Infinity, west: Infinity };
```

Comment:
not sure when would that be useful, but that's how it was working before: ` Math.min.apply(null, maxLat)` returns  Infinity if `maxLat == []`.